config.yaml.example

```diff
@@ -8,6 +8,9 @@ mqtt:
   availability_topic: lwt_topic
 
 manager:
+  sensor_config:
```

Comment:
Maybe it would be better to specify this as option per each worker.

Reply:
really? 🤔 i thought the device discovery is more a global feature, then a worker feature 🤔

Reply:
Maybe you are right, I'm not familiar with autodiscovery too much. From looking at json payloads, it looks like a lot of hardcoded values, so I was thinking it might be better to allow opt in per worker, instead of global on / off. Maybe thats too customizable, lets see how it will work like this :D

Reply:
that was my intension, keep it simple 😁
when i'm looking at my situation, i want the discovery feature for "everything" (for every worker, which implemented the needed function) or for nothing. 🙂